deploy/charts/arcadia/values.yaml

```diff
@@ -1,6 +1,10 @@
 global:
   oss:
     bucket: &default-oss-bucket "arcadia"
+  ## @param global.defaultVectorStoreType Defines the default vector database type, currently `chroma` and `pgvector` are available
+  ## When the option is `chroma`, it needs `chromadb.enabled` to be `true` as well to work.
+  ## When the option is `pgvector`, it needs `postgresql.enabled` to be `true` as well to work.
+  defaultVectorStoreType: pgvector
```

Comment:
chromadb is still enabled for now, can we update it to false if we'll use pgvector by default now?

Reply:
We have 2 separate configurations: `chromadb.enabled` and `postgresql.enable` which control the startup of chroma and postgresql respectively, and I think it's possible to change `chromadb.enabled` to `false` by default from the current version onwards.Any ideas? @nkwangleiGIT @bjwswang @0xff-dev

Reply:
@Abirdcfly  agree. We can focus on pgvector. 

Set chroma to false by default. 

Besides.. if chroma or pg enbaled,we should create a CR vectorstore as well.